server/app/views/applicant/TrustedIntermediaryDashboardView.java

```diff
@@ -259,34 +273,111 @@ private TrTag renderTIRow(AccountModel ti) {
+        .with(
+            div()
+                .withClass("usa-card__container")
+                .with(
+                    div()
+                        .withClasses("usa-card__header", "flex justify-between")
+                        .with(
+                            div(
+                                    div(
+                                            renderSubHeader(account.getApplicantName())
+                                                .withClass("usa-card__heading"),
+                                            u(renderEditClientLink(account.id)).withClass("ml-2"))
+                                        .withClass("flex"),
+                                    renderClientDateOfBirth(account))
+                                .withClass("flex-col"),
+                            renderIndexPageLink(account)),
+                    div()
+                        .withClasses("usa-card__body", "flex")
+                        .with(
+                            renderCardContactInfo(account).withClasses("w-2/5"),
+                            renderCardApplications(account).withClasses("ml-10 w-2/5"),
+                            renderCardNotes(account.getTiNote()).withClasses("ml-10 w-3/5"))));
+  }
+
+  private DivTag renderCardContactInfo(AccountModel account) {
+    Optional<ApplicantModel> newestApplicant = account.newestApplicant();
+    if (newestApplicant.isEmpty()) {
+      return div();
+    }
+    ApplicantData applicantData = newestApplicant.get().getApplicantData();
+    Optional<String> maybePhoneNumber = applicantData.getPhoneNumber();
+    String email = account.getEmailAddress();
+
+    return div(
+        label("Contact information").withFor("card_contact_info").withClass("whitespace-nowrap"),
+        div()
+            .condWith(
+                maybePhoneNumber.isPresent(),
+                div(
+                        Icons.svg(Icons.PHONE).withClasses("h-3", "w-3", "mr-1"),
+                        p(formatPhone(maybePhoneNumber.orElse(""))))
+                    .withClass("flex items-center"))
+            .condWith(
+                email != null && !email.isEmpty(),
+                div(Icons.svg(Icons.EMAIL).withClasses("h-3", "w-3", "mr-1"), p(email))
+                    .withClass("flex items-center"))
+            .withClass("text-xs")
+            .withId("card_contact_info"));
   }
 
-  private TdTag renderUpdateClientInfoCell(Long accountId) {
-    return td().with(
-            new LinkElement()
-                .setId("edit-client")
-                .setText("Edit")
-                .setHref(
-                    controllers.ti.routes.TrustedIntermediaryController.editClient(accountId).url())
-                .asAnchorText())
-        .withClasses(BaseStyles.TABLE_CELL_STYLES, "pr-12");
+  private String formatPhone(String phone) {
+    try {
+      Phonenumber.PhoneNumber phoneNumber = PHONE_NUMBER_UTIL.parse(phone, "US");
```

Comment:
Nit- Can you use this this from the TIService class? I believe the country code is declared as a static string

Reply:
Good idea.  I've made the change.